sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -712,7 +712,7 @@ def __repr__(self):
                                      self.grapheme_length, self.warnings)[:1024]
 
 
-class SentimentConfidenceScorePerLabel(DictMixin):
+class SentimentConfidenceScores(DictMixin):
     """The confidence scores (Softmax scores) between 0 and 1 across all sentiment
     labels: positive, neutral, negative. Higher values indicate higher confidence.
```

Comment:
since we're dropping the "per label" part let's remove that from the docstring too